cibyl/models/attribute.py

```diff
@@ -15,12 +15,49 @@
 """
 from dataclasses import dataclass
 
+from cibyl.cli.argument import Argument
+
 
 @dataclass
 class AttributeValue():
     """Represents the value used by the attributes of the different models"""
 
     name: str
-    type: object
+    attr_type: object
     value: object
-    arguments: list
+    arguments: list[Argument]
+
+
+class AttributeListValue(AttributeValue):
+    """Represents a list of AttributeValue objects"""
+
+    def __init__(self, name, arguments=None, attr_type=None, value=None):
+
+        super(AttributeListValue, self).__init__(
+            name=name, arguments=arguments, attr_type=attr_type)
+
+        if isinstance(value, list):
+            self.value = value
+        if value is None:
+            self.value = []
+
+    def append(self, item):
+        """
+        :param item:
+        :return:
+        """
+        self.value.append(item)
+
+    def __getitem__(self, index):
+        """
+        :param index:
+        :return:
+        """
+        return self.value[index]
+
+    def __lt__(self, other):
+        """
+        :param other:
+        :return:
+        """
+        return self < other
```

Comment:
I believe that if the operator is to be implemented, then it should be done relative to the attribute's value in some way. Besides, wouldn't this expression lead to a recursive call?

Reply:
Thanks, removed it in the meantime.